book/src/contributing/using-dev-container.md

```diff
@@ -0,0 +1,48 @@
+# PRQL in a Dev Container
+
+_This is a first-cut description of using Dev Containers. It could use lots more info about installing and using Dev Containers on VS Code._
+
+A Dev Container is a way to package a number of "developer tools" (compilers, bundlers, package managers, loaders, etc.) into a single object.
+This is helpful when many people want to contribute to a project: each person only has to install the (single) Dev Container on their own machine to start working.
+By definition, the Dev Container has a consistent set of tools that are known to work together.
+This avoids a fuss with finding the proper versions of each of the build tools.
+
+To use a Dev Container, you must install these pre-requisites:
+
+* [Docker](https://www.docker.com/) (follow any of the instructions on the internet for your computer)
+* [VS Code](https://code.visualstudio.com/) Interactive development environment
+* [Dev Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension for VS Code
```

Comment:
These are ways to use VS Code Dev Containers and are not required dependencies for Dev Containers.\
For example, if you use GitHub Codespaces, you need nothing more than a browser.
It would be great if we could express that.

See all supporting tools. https://containers.dev/supporting

Reply:
See the updates. (I don't know much about Codespaces, so will leave that description to others.)